lib/plugins/html/templates/url/waterfall/fetchHAR.pug

```diff
@@ -15,11 +16,27 @@ script(type='text/javascript').
     selectedPage: #{runIndex || 0}
   };
 
-  if (window["fetch"]) {
-      window["fetch"]('data/browsertime.har')
-      .then(f => f.json().then(j => {
-        var har = j.log;
-        var perfCascadeSvg = perfCascade.fromHar(har, options);
-        outputHolderEl.appendChild(perfCascadeSvg);
-      }))
-  }
+if options.gzipHAR
+    script(type='text/javascript').
+      if (window["fetch"]) {
+          window["fetch"]('data/browsertime.har.gz')
+          .then(f => f.arrayBuffer().then(arrayBuffer => {
+            var byteArray = new Uint8Array(arrayBuffer);
+            var gunzip = new Zlib.Gunzip(byteArray);
+            var decompressedArray = gunzip.decompress();
+            var string = new TextDecoder("utf-8").decode(decompressedArray);
+            var har = (JSON.parse(string)).log;
+            var perfCascadeSvg = perfCascade.fromHar(har, options);
+            outputHolderEl.appendChild(perfCascadeSvg);
+          }))
+      }
+else
+    script(type='text/javascript').
+      if (window["fetch"]) {
+       window["fetch"]('data/browsertime.har')
+       .then(f => f.json().then(j => {
```

Comment:
Change the parenthesis (adding one here, remove one at the end), to avoid unneeded nesting of the promise flow. Use
```
...
.then(f => f.json())
.then(har => { ...})
```